src/sagemaker/jumpstart/utils.py

```diff
@@ -1128,19 +1132,66 @@ def get_jumpstart_user_agent_extra_suffix(
     else:
         headers = f"{sagemaker_python_sdk_headers} {jumpstart_specific_suffix}"
 
+    if config_name:
+        headers = f"{headers} {config_specific_suffix}"
+
     return headers
 
 
+def get_top_ranked_config_name(
```

Comment:
non-blocking: are we assuming `overall` ranking in this method? Wouldn't we need to be provided which ranking array to use?

Reply:
That's an excellent point. How about I make it an optional argument, the ranking name, with the default value being overall?